internal/website/content/howto/runtimevar/_index.md

```diff
@@ -175,6 +175,14 @@ https://godoc.org/gocloud.dev/runtimevar/awsparamstore#OpenVariable
 
 ### etcd {#etcd}
 
+*NOTE*: Support for `etcd` has been temporarily dropped until they sort out
+their Go module dependency problems. See
+https://github.com/google/go-cloud/issues/2914.
+
+You can use `runtimevar.etcd` in Go CDK version `v0.20.0` or earlier.
+
+<!--
```

Comment:
Consider just erasing this for now - should be easy to pull up from git later on.

Reply:
Done.